src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -302,7 +303,7 @@ private void updateUsers()
     {
         users.clear();
         users.addAll(townHall.getColony().getPlayers().values());
-        users.sort(Comparator.comparing(Permissions.Player::getRank, Permissions.Rank::compareTo));
+        users.sort(Comparator.comparing(Permissions.Player::getRank, Rank::compareTo));
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready